moveit_planners/chomp/chomp_interface/include/chomp_interface/chomp_interface.h

```diff
@@ -48,7 +48,7 @@ MOVEIT_CLASS_FORWARD(CHOMPInterface);
 class CHOMPInterface : public chomp::ChompPlanner
 {
 public:
-  CHOMPInterface();
+  CHOMPInterface(const ros::NodeHandle& nh = ros::NodeHandle("~"));
```

Comment:
`nh_` gets used in `CHOMPInterface::loadParams()`. Previously it was using `NodeHandle()` by default which means parameters are read from global name space. This change moves the parameters to `~` which typically means `/move_group/*`. It's a bit more structured that way. The example file in https://github.com/ros-planning/moveit_resources/pull/16 uses this location already.